src/app/zones/components/ZoneEgressDetails.vue

```diff
@@ -85,38 +101,39 @@ import { computed, PropType } from 'vue'
 
 import AccordionItem from '@/app/common/AccordionItem.vue'
 import AccordionList from '@/app/common/AccordionList.vue'
-import DefinitionList from '@/app/common/DefinitionList.vue'
-import DefinitionListItem from '@/app/common/DefinitionListItem.vue'
+import DefinitionCard from '@/app/common/DefinitionCard.vue'
 import EnvoyData from '@/app/common/EnvoyData.vue'
+import StatusBadge from '@/app/common/StatusBadge.vue'
 import SubscriptionDetails from '@/app/common/subscriptions/SubscriptionDetails.vue'
 import SubscriptionHeader from '@/app/common/subscriptions/SubscriptionHeader.vue'
 import TabsWidget from '@/app/common/TabsWidget.vue'
 import TextWithCopyButton from '@/app/common/TextWithCopyButton.vue'
 import type { ZoneEgressOverview } from '@/types/index.d'
 import { useI18n } from '@/utilities'
+import { getItemStatusFromInsight } from '@/utilities/dataplane'
 
 const { t } = useI18n()
 
 const TABS = [
   {
     hash: '#overview',
-    title: 'Overview',
+    title: t('zone-egresses.routes.item.tabs.overview'),
   },
   {
     hash: '#insights',
-    title: 'Zone Egress Insights',
+    title: t('zone-egresses.routes.item.tabs.insights'),
   },
   {
     hash: '#xds-configuration',
-    title: 'XDS Configuration',
+    title: t('zone-egresses.routes.item.tabs.xds_configuration'),
```

Comment:
nitpick: why do we mix `-` and `_`? shouldn't it both be `zone_egresses` and `xds_configuration` or `zone-egresses` and `xds-configuration`

Reply:
I’m starting to only use snakecase as that will be what we’ll standardize on. However, to minimize changes, I’m not changing existing keys because that would require also changing all their uses.